store/store.go

```diff
@@ -121,15 +138,42 @@ func (t *Txn) SetCurrent(key, name string, global, def bool) error {
 	h := toHash(key)
 
 	if def {
-		if err := ioutils.AtomicWriteFile(filepath.Join(t.s.root, "defaults", h), []byte(name), 0600); err != nil {
+		if err := ioutils.AtomicWriteFile(filepath.Join(t.s.root, defaultsDir, h), []byte(name), 0600); err != nil {
 			return err
 		}
 	} else {
-		os.RemoveAll(filepath.Join(t.s.root, "defaults", h)) // ignore error
+		os.RemoveAll(filepath.Join(t.s.root, defaultsDir, h)) // ignore error
 	}
 	return nil
 }
 
+func (t *Txn) UpdateLastActivity(ng *NodeGroup) error {
+	return ioutils.AtomicWriteFile(filepath.Join(t.s.root, activityDir, ng.Name), []byte(fmt.Sprintf("%d", time.Now().Unix())), 0600)
```

Comment:
Any reason for using the Unix timestamp in the file? Could we instead use `time.RFC3339` as the format?

Would prefer something that's vaguely human-readable if possible, unless we have precedent for a different format?